pinot-server/src/main/java/org/apache/pinot/server/api/resources/InstanceResource.java

```diff
@@ -73,4 +75,20 @@ public List<String> getInstanceTags() {
     }
     return Collections.emptyList();
   }
+
+  @GET
+  @Path("pools")
+  @ApiOperation(value = "Tenant pools for current instance")
+  @ApiResponses(value = {
+      @ApiResponse(code = 200, message = "Success"), @ApiResponse(code = 500, message = "Internal server error")
+  })
```

Comment:
1) can you add the javadoc to explain return result types? i.e.,The summary in the PR looks good.
2) Can you add a Pinot doc reference to the definition of pool in Pinot?